nextflow_schema.json

```
@@ -297,30 +297,22 @@
                     "fa_icon": "fas fa-adjust"
                 },
-                "annotation_prokka_centre": {
-                    "type": "string",
-                    "description": "Sequencing centre ID.",
-                    "fa_icon": "fas fa-map-marker-alt",
-                    "help_text": "Add the sequencing center ID used in generating the raw sequences. This flag is typically requested in combination with the `--compliant` flag when contigs need to be renamed due to non-conforming contig headers. For more information please check Prokka [documentation](https://github.com/tseemann/prokka). \n\n> Modifies tool parameter(s):\n> - Prokka: `--centre`"
-                },
                 "annotation_prokka_compliant": {
                     "type": "boolean",
                     "fa_icon": "far fa-check-circle",
                     "description": "Force contig name to Genbank/ENA/DDJB naming rules.",
                     "help_text": "Force the contig headers to conform to the Genbank/ENA/DDJB contig header standards. This is activated in combination with `--centre [X]` when contig headers supplied by the user are non-conforming and therefore need to be renamed before Prokka can start annotation. This flag activates  `--genes --mincontiglen 200`. For more information please check Prokka [documentation](https://github.com/tseemann/prokka). \n\n> Modifies tool parameter(s):\n> - Prokka: `--compliant`"
                 },
-                "annotation_prokka_locustag": {
-                    "type": "string",
-                    "default": "Prokka",
-                    "fa_icon": "fas fa-tags",
-                    "description": "Assign the locus tag for the contig header.",
-                    "help_text": "Assign a special name to the contig. This is used when a specific group of samples are run in a batch. For more information please check Prokka [documentation](https://github.com/tseemann/prokka). \n\n> Modifies tool parameter(s):\n> - Prokka: `--locustag`"
-                },
                 "annotation_prokka_addgenes": {
                     "type": "boolean",
                     "fa_icon": "fas fa-dna",
                     "description": "Add the gene features for each CDS hit.",
                     "help_text": "For every CDS annotated, this flag adds the gene that encodes for that CDS region. For more information please check Prokka [documentation](https://github.com/tseemann/prokka). \n\n> Modifies tool parameter(s):\n> - Prokka: `--addgenes`"
+                },
+                "annotation_prokka_retaincontigheaders": {
+                    "type": "boolean",
```

```suggestion
                    "type": "boolean",
                    "fa_icon":"fas fa-remove-format",          
```
how about this icon https://fontawesome.com/v5/icons/remove-format?f=classic&s=solid